torch/_export/constraints.py

```diff
@@ -1,10 +1,23 @@
 from typing import Optional
 
+import torch
+from torch import SymInt
 from torch._dynamo import allow_in_graph
-from torch.fx.experimental.symbolic_shapes import constrain_range
+from torch.fx.experimental.symbolic_shapes import constrain_range_non_symint
 from torch.utils._sympy.value_ranges import ValueRangeError
 
 
+def _constrain_range(symbol, min: Optional[int] = None, max: Optional[int] = None):
+    if not isinstance(symbol, SymInt):
+        # This is needed to check if an input value (as a real numeric value, or
+        # an value derived from numeric input) for exporting is within range or
+        # not.
+        constrain_range_non_symint(symbol, min=min, max=max)
+        return
+
+    torch.sym_constrain_range(symbol, min, max)
```

Comment:
@tugsbayasgalan: do we actually want to manually suppress this? I didn't see there is an Sym* version of Scalar so I guess the way to support this is to have overloads for int (SymInt in native_functions.yaml) or float (SymFloat in native_functions.yaml)?